src/libopensc/card-openpgp.c

```diff
@@ -88,31 +88,21 @@ static struct sc_card_driver pgp_drv = {
 
 
 static pgp_ec_curves_t	ec_curves[] = {
-	{{{1, 2, 840, 10045, 3, 1, 7, -1}}, 256,
-		{{0x2a, 0x86, 0x48, 0xce, 0x3d, 0x03, 0x01, 0x07, -1}}}, /* ansiX9p256r1 */
-	{{{1, 3, 132, 0, 34, -1}}, 384,
-		{{0x2b, 0x81, 0x04, 0x00, 0x22, -1}}}, /* ansiX9p384r1 */
-	{{{1, 3, 132, 0, 35, -1}}, 521,
-		{{0x2b, 0x81, 0x04, 0x00, 0x23, -1}}}, /* ansiX9p521r1 */
-	{{{1, 3, 36, 3, 3, 2, 8, 1, 1, 7, -1}}, 256,
-		{{0x2b, 0x24, 0x03, 0x03, 0x02, 0x08, 0x01, 0x01, 0x07, -1}}}, /* brainpoolP256r1 */
-	{{{1, 3, 36, 3, 3, 2, 8, 1, 1, 11, -1}}, 384,
-		{{0x2b, 0x24, 0x03, 0x03, 0x02, 0x08, 0x01, 0x01, 0x0b, -1}}}, /* brainpoolP384r1 */
-	{{{1, 3, 36, 3, 3, 2, 8, 1, 1, 13, -1}}, 512,
-		{{0x2b, 0x24, 0x03, 0x03, 0x02, 0x08, 0x01, 0x01, 0x0d, -1}}}, /* brainpoolP512r1 */
-	{{{-1}}, 0, {{0x0}}} /* This entry must not be touched. */
+	{{{1, 2, 840, 10045, 3, 1, 7, -1}}, 256}, /* ansiX9p256r1 */
+	{{{1, 3, 132, 0, 34, -1}}, 384}, /* ansiX9p384r1 */
+	{{{1, 3, 132, 0, 35, -1}}, 521}, /* ansiX9p521r1 */
+	{{{1, 3, 36, 3, 3, 2, 8, 1, 1, 7, -1}}, 256}, /* brainpoolP256r1 */
+	{{{1, 3, 36, 3, 3, 2, 8, 1, 1, 11, -1}}, 384}, /* brainpoolP384r1 */
+	{{{1, 3, 36, 3, 3, 2, 8, 1, 1, 13, -1}}, 512}, /* brainpoolP512r1 */
+	{{{-1}}, 0} /* This entry must not be touched. */
 };
 
 static pgp_ec_curves_t	ec_curves_gnuk[] = {
-	{{{1, 2, 840, 10045, 3, 1, 7, -1}}, 256,
-		{{0x2a, 0x86, 0x48, 0xce, 0x3d, 0x03, 0x01, 0x07, -1}}}, /* ansiX9p256r1 */
-	{{{1, 3, 132, 0, 10, -1}}, 256,
-		{{0x06, 0x05, 0x2B, 0x81, 0x04, 0x00, 0x0A, -1}}}, /* secp256k1 */
-	/*{{{1, 3, 6, 1, 4, 1, 3029, 1, 5, 1, -1}}, 256,
-		{{0x0A, 0x2B, 0x06, 0x01, 0x04, 0x01, 0x97, 0x55, 0x01, 0x05, 0x01, -1}}}, //cv25519
-	{{{1, 3, 6, 1, 4, 1, 11591, 15, 1, -1}}, 256,
-		{{0x09, 0x2B, 0x06, 0x01, 0x04, 0x01, 0xDA, 0x47, 0x0F, 0x01, -1}}}, // ed25519 */
-	{{{-1}}, 0, {{0x0}}} /* This entry must not be touched. */
+	{{{1, 2, 840, 10045, 3, 1, 7, -1}}, 256}, /* ansiX9p256r1 */
+	{{{1, 3, 132, 0, 10, -1}}, 256}, /* secp256k1 */
```

Comment:
clang-format suggested the following change:


```
-	{{{1, 3, 132, 0, 10, -1}}, 256}, /* secp256k1 */
+	{{{1, 3, 132, 0, 10, -1}}, 256},          /* secp256k1 */
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-openpgp.c -->